src/components/CalendarDay.jsx

```diff
@@ -58,7 +58,11 @@ class CalendarDay extends React.PureComponent {
     const { isFocused, tabIndex } = this.props;
     if (tabIndex === 0) {
       if (isFocused || tabIndex !== prevProps.tabIndex) {
-        this.buttonRef.focus();
+        requestAnimationFrame(() => {
```

Comment:
do we already reference rAF elsewhere in the repo, or is this dependency a potential breaking change?

Reply:
This looks like the first usage

Reply:
any chance we could use https://npmjs.com/raf instead of the global?